src/index.ts

```diff
@@ -432,29 +532,32 @@ class Oas {
    * object and another one for `operation`. This differs from `getOperation()` in that it does not return an instance
    * of the `Operation` class.
    *
-   * @param {String} url
-   * @param {String} method
-   * @return {(Object|undefined)}
+   * @param url A full URL to look up.
+   * @param method The cooresponding HTTP method to look up.
+   * @returns An object containing matches that were discovered in the API definition.
    */
-  findOperation(url, method) {
+  findOperation(url: string, method: RMOAS.HttpMethods): PathMatch {
     const annotatedPaths = this.findOperationMatches(url);
     if (!annotatedPaths) {
       return undefined;
     }
 
-    const includesMethod = filterPathMethods(annotatedPaths, method);
-    if (!includesMethod.length) return undefined;
-    return findTargetPath(includesMethod);
+    const matches = filterPathMethods(annotatedPaths, method) as Array<{
```

Comment:
This is happening because of some funky stuff with `PathMatch.p` sometimes being a `PathsObject` and other times being an `OperationObject`. Since all the tests here still work I'm going to follow up this PR with a better fix for this.